CHANGELOG.md

```diff
@@ -25,8 +25,12 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 - Addition of autoSelect prop to React SelectElement allows for auto-selection of only available select option (PR #6156)
 - An `AcquisitionDate` field has been added to the `files` table (PR #6892)
 - The default value of the ScannerID field of the mri_protocol table is now NULL instead of 0 (PR #7496).
+- Pending accounts in Dashboard are now includes DDC users (PR #7054)
+- Subproject filter added to Behavioural QC module (PR #7430)
 #### Bug Fixes
 - The default value of the `ScannerID` field of the `mri_protocol` table is now `NULL` instead of `0`. This means that if a protocol is valid on all the study's scanners, then `ScannerID` of the protocol should be set to `NULL` (PR #7496)
+- In Candidate Parameters, fix caveat emptor flag being editable without a permission to edit candidate profiles(PR #7047)
+- Candidate library now allows a null sex in the select() function to accommodate scanner candidates. This prevents an error from being thrown in the candidate parameters module. (PR #7058)
```

Comment:
not necessary, seems like a bug introduced by the same version

Reply:
Are you talking about line 32 or 33 or both?

Reply:
if the bug on line 32 existed in version 23, keep it, if it was introduced and fixed in 24, remove it

Reply:
Looks like it did not exist in 23 I will remove

Reply:
did the bug on line 41 exist in 23 ? if not remove it as well